hs-bindgen-libclang/src/HsBindgen/Clang/Aux/Fold.hs

```diff
@@ -1,13 +1,12 @@
 -- | Higher-level bindings for traversing the API
-module HsBindgen.Clang.Core.Util.Fold (
+module HsBindgen.Clang.Aux.Fold (
```

Comment:
Are you going to rename this module in every patch? :)

Reply:
Possibly? :)

Reply:
Ah, sorry, I'm being slow on the uptake. Thank you for the clue :) Will fix.

Reply:
Have renamed `Aux` back to `Util`, to avoid the name `Aux` which is illegal on Windows. The fact that only the `Classification.hs` was reported as a problem threw me off the scent (but of course in hindsight it was just the _first_ problematic file).

Reply:
I opened a cabal-install issue https://github.com/haskell/cabal/issues/10295, that would helped figure out the issue. (Without CI we wouldn't find it until possibly too late).

Reply:
Ah, good idea, thanks :)